docs/source/methods/depletion.rst

```diff
@@ -257,3 +257,70 @@ choose one of two methods for estimating the heating rate, including:
 
 The method for normalization can be chosen through the ``normalization_mode``
 argument to the :class:`openmc.deplete.CoupledOperator` class.
+
+----------------
+MSR Capabilities
+----------------
+
+A nice feature of Molten Salt Reactors (MSRs), and in general of
+*any*  liquid fuel reactor, is the capability to continuously reprocess the fuel
+to remove fission products or to feed fresh fuel into the system.
+
+Removal rate
+------------
+
+A removal rate as defined here is the rate at which nuclides are
+continuously removed from a material. Mathematically, it
+can be thought as an additional proportional term :math:`\mathbf{T}`
+to the depletion equation, which can be written as:
+
+.. math::
+
+  \begin{aligned}\frac{dN_i(t)}{dt} = &\underbrace{\sum\limits_j f_{j\rightarrow i}
+  \int_0^\infty dE  \; \sigma_j (E,t) \phi(E,t) N_j(t)  - \int_0^\infty dE \; \sigma_i(E,t)
+  \phi(E,t) N_i(t)}_\textbf{R} \\
+  &+ \underbrace{\sum_j \lambda_{j\rightarrow i} N_j(t) + \lambda_{i\rightarrow j} N_i(t)}_\textbf{D} \\
+  &- \underbrace{\epsilon_i \lambda_i N_i(t)}_\textbf{T} \end{aligned}
+
+where the transmutation term :math:`\mathbf{R}`, the decay term :math:`\mathbf{D}` and the new removal term :math:`\mathbf{T}`
+have been grouped together so that
+:math:`\mathbf{A} = \mathbf{R}+\mathbf{D}-\mathbf{T}`.
+
+the removal efficiency :math:`\epsilon_i` and the removal rate coefficient :math:`\lambda_i`
+define the continuous removal of the nuclide :math:`i`, which behaviour is similar to radioactive decay.
+For simplicity, :math:`\epsilon_i` and :math:`\lambda_i`
+can be combined together in one single user-defined parameter that is defined again as :math:`\lambda_i`.
```

Comment:
Does the user ever use both $\epsilon_i$ and $\lambda_i$ in the implementation? If not, maybe that could be a functionality that is added? It seems useful.

Reply:
Since it is just two constants multiplying each other, to me makes more sense to combine them in one.

Reply:
I think being able to tweak the efficiency on-the-fly instead of making users do that would be a nice feature to have. The default efficiency could be 1. But it's not critical.